src/Microsoft.Diagnostics.EventFlow.Inputs.DiagnosticSource/Configuration/DiagnosticSourceConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.Diagnostics.EventFlow.Configuration
+{
+    public sealed class DiagnosticSourceConfiguration
```

Comment:
Please leave the classes unsealed if possible. I know some recommend to seal all classes by default, but we do not do that in EventFlow code.